examples/basics/plotting/grid-basic.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
```

Comment:
name it `grid_basic` instead of `grid-basic`. It's better to use underscores than hyphens in python filenames to facilitate importing